spec/controllers/welcome_controller_spec.rb

```diff
@@ -4,8 +4,9 @@
 
   describe "GET #index" do
     it "returns http success" do
```

Comment:
need to change test name or remove it comepletely

Reply:
renamed - welcome page should redirect to sign up page